Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -1745,6 +1745,95 @@ public static void ConnectAppleAccountByRestSignIn(string authorizationCode, Act
             LootLockerServerRequest.CallAPI(forPlayerWithUlid, endpoint, LootLockerEndPoints.connectProviderToAccount.httpMethod, data, (response) => { LootLockerResponse.Deserialize(onComplete, response); });
         }
 
+        /// <summary>
+        /// Connect a Twitch Account to the currently logged in LootLocker account allowing that Twitch account to start sessions for this player
+        /// IMPORTANT: If you are using multiple users, be very sure to pass in the correct `forPlayerWithUlid` parameter as that will be the account that the Twitch account is linked into
+        /// </summary>
+        /// <param name="authorizationCode">The Authorization Code from Twitch sign in</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        /// <param name="forPlayerWithUlid">Optional : Execute the request for the specified player. If not supplied, the default player will be used.</param>
+        public static void ConnectTwitchAccount(string authorizationCode, Action<LootLockerAccountConnectedResponse> onComplete, string forPlayerWithUlid = null)
```

Comment:
This might be a dumb question, but how does the user obtain the `authorizationCode` value?